content/tutorials/use-simple-xrp-payments/send-xrp.md

```diff
@@ -173,7 +196,7 @@ Now that you have a signed transaction, you can submit it to an XRP Ledger serve
 
 - **JavaScript:** Use the [submit() method](rippleapi-reference.html#submit) to submit a transaction to the network. Use the [`getLedgerVersion()` method](rippleapi-reference.html#getledgerversion) to get the latest validated ledger index.
 - **Python:** Use the [xrpl.transaction.submit_transaction() method](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.transaction.html#xrpl.transaction.submit_transaction) to submit a transaction to the network. Use the [xrpl.ledger.get_latest_validated_ledger_sequence() method](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.ledger.html#xrpl.ledger.get_latest_validated_ledger_sequence) to get the latest validated ledger index.
-
+- **Java:** Use the [XrplClient.submit(SignedTransaction>) method](https://javadoc.io/doc/org.xrpl/xrpl4j-client/latest/org/xrpl/xrpl4j/client/XrplClient.html) to submit a transaction to the network. Use the `XrplClient.ledger()` method to get the latest validated ledger index.
```

Comment:
The `XrplClient.ledger()` method should probably be linked to <https://javadoc.io/doc/org.xrpl/xrpl4j-client/latest/org/xrpl/xrpl4j/client/XrplClient.html#ledger(org.xrpl.xrpl4j.model.client.ledger.LedgerRequestParams)>, right?